hal/src/boron/cellular_hal.cpp

```diff
@@ -96,8 +96,9 @@ hal_net_access_tech_t fromCellularAccessTechnology(CellularAccessTechnology rat)
         return NET_ACCESS_TECHNOLOGY_UTRAN;
     case CellularAccessTechnology::LTE:
     case CellularAccessTechnology::E_UTRAN:
-    case CellularAccessTechnology::EC_GSM_IOT: // FIXME
         return NET_ACCESS_TECHNOLOGY_LTE;
+    case CellularAccessTechnology::EC_GSM_IOT:
```

Comment:
I think we may want to move `CellularAccessTechnology::E_UTRAN:` down here as well in the  NET_ACCESS_TECHNOLOGY_LTE_IOT result, since it's Cat-NB1.

Reply:
I was reading and reading, and the EUTRA has been around so long, I couldn't tack it down as to what it was. I will gladly move it down.

Reply:
Cat-M1 is somewhat hard to know based on name alone, since there are some footnotes in the R4 AT command manual, but Cat-NB1 is a little clearer:

### COPS
![Screen Shot 2019-04-29 at 12 51 46 PM](https://user-images.githubusercontent.com/2249224/56916097-bdd7bf00-6a7d-11e9-9c1c-7e7635959fe6.png)

![Screen Shot 2019-04-29 at 12 53 27 PM](https://user-images.githubusercontent.com/2249224/56916123-cd570800-6a7d-11e9-8d56-514532672971.png)

Reply:
Fix is in.